app/views/layouts/_ae_resolve_options.html.haml

```diff
@@ -85,19 +85,6 @@
             :javascript
               miqInitSelectPicker();
               miqSelectPickerEvent('target_id', "#{url}")
-- if ae_sim_form
```

Comment:
@syncrou this is the code for actual automate simulation screen, do we need to remove Execute Methods checkbox form there as well?